Fhi.Smittestopp.Verification.Server/Startup.cs

```diff
@@ -55,7 +56,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IAccountService, AccountService>();
 
             services.AddDomainServices(Configuration.GetSection("common"));
-            services.AddMockMsisLookup();
+
+            services.AddMsisLookup(Configuration.GetSection("msis"));
```

Comment:
An idea to name the config section 'MsisConfig' so that the nameof(MsisConfig) could be used here as a more strongly typed 'one fact in one place' name instead of 'msis'? Also applies to the other config sections, of course. No objections, just asking 👍

Reply:
I guess this is more a personal preference. I guess after getting bitten once due to a careless class rename breaking a configuration, I am hesitant to use nameof in this case, but I could be convinced to switch.

Reply:
I agree, it's a personal preference. And I certainly see your point here, getting bitten due to a class rename is very painful. Good point.